metric/cpu/metrics_windows.go

```diff
@@ -26,13 +26,102 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/elastic/elastic-agent-libs/helpers/windows/pdh"
 	"github.com/elastic/elastic-agent-libs/opt"
 	"github.com/elastic/elastic-agent-system-metrics/metric/system/resolve"
 	"github.com/elastic/gosigar/sys/windows"
 )
 
+var (
+	processorInformationCounter = "\\Processor Information(%s)\\%s"
```

Comment:
Doing some searching, I think these names might be localized to the language of the Windows installs. See https://stackoverflow.com/questions/5581859/retrieve-performance-counter-value-in-a-language-independent-way for example. It looks like the more common ones can be identified by an ID instead of a locale dependent string.

Let's make sure we test this on a non-english version of Windows. We have had problems in the past on the French and German language installs of Windows.

See https://github.com/elastic/elastic-agent/issues/3960#issuecomment-1877700201 for example to get started.

Reply:
Yup. I recently had same discussion with @leehinman on https://github.com/elastic/elastic-agent-libs/pull/254#discussion_r1856899234.

I updated the functions to add counters in language neutral manner. See [here](https://github.com/VihasMakwana/elastic-agent-libs/blob/0d290c679f51bcf5f7d74a42ddf8e36a4ab4f878/helpers/windows/pdh/pdh_query_windows.go#L98-L103).

> Let's make sure we test this on a non-english version of Windows. We have had problems in the past on the French and German language installs of Windows.

Will do it. Thanks!

Reply:
Nice! Thank you @leehinman for also being haunted by this problem.